### ISSUE TITLE
[Suggestion]More examples

### ISSUE THREAD

**Issue by sarandafl** [requests/requests-oauthlib#262]
As a suggestion, could we have more examples? Since this can get quite complicated, someone might just need the right example to get started.

I am also currently interested in this one over [here](http://stackoverflow.com/questions/42213153/access-lovoo-api-using-python), but also can't get it to work.

Most of the examples use oauth2, even though oauth1 is still very much in use.

**Comment by Lukasa** [comment]
I completely agree. Anyone who'd like to commit code examples should feel free to propose pull requests to do so!
